.ci/scripts/check_lint.go

```diff
@@ -36,30 +39,33 @@ var ignoreWarnings = []string{
 var ignoreWarningsRe = regexp.MustCompile(strings.Join(ignoreWarnings, "|"))
 
 func main() {
+	log.SetFlags(0)
 	flag.Parse()
 
-	out, err := exec.Command("go", "get", "-u", "github.com/golang/lint/golint").Output()
+	goGet := exec.Command("go", "get", "-u", "golang.org/x/lint/golint")
+	goGet.Env = os.Environ()
+	goGet.Env = append(goGet.Env, "GO111MODULE=off")
+	out, err := goGet.Output()
 	if err != nil {
-		fmt.Fprintln(os.Stderr, "error", err)
-		os.Exit(1)
+		log.Fatalf("failed to %v: %v", strings.Join(goGet.Args, " "), err)
 	}
 
-	golint := exec.Command("golint", flag.Args()...)
+	golint := exec.Command(filepath.Join(build.Default.GOPATH, "bin", "golint"),
+		flag.Args()...)
 	golint.Env = os.Environ()
 	golint.Env = append(golint.Env, "GOOS=windows")
 	out, err = golint.Output()
 	if err != nil {
-		fmt.Fprintln(os.Stderr, "error", err)
-		os.Exit(1)
+		log.Fatalf("failed to %v: %v", strings.Join(golint.Args, " "), err)
 	}
 
 	out, err = filterIgnores(out)
 	if err != nil {
-		fmt.Fprintln(os.Stderr, "error", err)
-		os.Exit(1)
+		log.Fatal(err)
 	}
 
 	if len(out) > 0 {
+		log.Println("There are golint warnings.")
```

Comment:
Why not use log.Fatalf here too?

Reply:
I wanted to write the error message to stderr, then continue to pass through the output of goimports to stdout, then `exit 1`.